### PR TITLE
fix1578 & Related Docs

### PR DESCRIPTION
**Pull request by mDuo13**
- Documents the impact of the fix1578 amendment introduced in https://github.com/ripple/rippled/pull/2584
- Fixes DOC-2099.
- Clarifies some rippling details to help avoid some "gotchas" about the NoRipple setting
- Re-add a missing note about how crossing your own offers automatically cancels the old ones.

### CODE REVIEW

File: content/concepts/issued-currencies/rippling.md [hunk 1]
```diff
@@ -52,7 +53,9 @@ The NoRipple flag makes certain paths invalid, so that they cannot be used to ma
 
 ## The DefaultRipple Flag
 
-The DefaultRipple flag is an account setting that enables rippling on all trust lines by default. Gateways and other currency issuers MUST enable this flag for other addresses to be able to send those the currencies among themselves.
+The DefaultRipple flag is an account setting that enables rippling on all incoming trust lines by default. Gateways and other currency issuers MUST enable this flag for other addresses to be able to send those the currencies among themselves.
```
Comment (jhaaaa) [inline content/concepts/issued-currencies/rippling.md]: Nit: What does "those" refer to? Is there a missing or extra word there?

>to be able to send *those* the currencies among themselves.

**Review by jhaaaa — APPROVED** [review]
Left a note about one possible nit edit - other than that, looks great - thank you!

File: content/concepts/issued-currencies/rippling.md (hunk 1 above)
Comment (mDuo13) [reply content/concepts/issued-currencies/rippling.md]: Yeah, I think that's an old typo. Should be either "those currencies" or "the currencies" not "those the currencies"

Comment (scottschurr) [reply content/concepts/issued-currencies/rippling.md]: Good eyes, @jhaaaa.  I missed that when I read it over.

**Review by scottschurr — APPROVED** [review]
:+1: Looks good to me.

**Comment by mDuo13** [comment]
The only broken link is to the 1.2.0 release, so this is :heavy_check_mark:

**Pull request merged**